job/dhcp.go

```diff
@@ -64,7 +64,9 @@ func (j Job) configureDHCP(rep, req *dhcp4.Packet) bool {
 		}
 
 		isSNP := dhcp.IsSNP(req)
-		j.Info("dhcp arch is x64 EFI, using SNP")
+		if isSNP {
+			j.Info("dhcp arch is x64 EFI, using SNP")
+		}
```

Comment:
Actually I don't think we need this here, the file name is logged already normally not sure what benefit this brings. If you do want to log anyway I'd say `setPXEFilename`'s `isPacket` is a better spot.

Reply:
i'll remove.  As agreed its not needed really

Reply:
great, thanks!